### DISCUSSION TITLE
Use fluetvalidation with blazorise controls

### DISCUSSION THREAD

**Discussion by yangetze** [Megabit/Blazorise#3235] [Ideas]
Hi!!! 
I would like to use fluentvalidation with blazorise controls. We're using bootstrap. 
In my team with implement fluentvalidation, but only works with pure html, but look n feel is pretty ugly. 
![imagen](https://user-images.githubusercontent.com/23171333/146217233-991dd980-cf5c-4429-8a25-663a71b3ed24.png)
I searched on the internet, but all the examples use pure HTML and I didn't find another implementation :( . 

We'd like to use fluentvalidation with the look n feel of blazorise
![imagen](https://user-images.githubusercontent.com/23171333/146218410-f3b8dc7f-6fa3-4c01-8528-f837bd0d3462.png)

Is this possible? Or is there another option? 

We chose fluentvalidation because we like to have all businesses rules in one place.

**Comment by stsrki** [comment]
We have it in the plan but we still don't have official FleuntValidation support. You guys can look at #1971 for inspiration and make your own version.
